.pre-commit-config.yaml

```diff
@@ -6,6 +6,7 @@ repos:
     rev: 22.12.0
     hooks:
     - id: black
+      exclude: opensafely/_vendor/
```

Comment:
We can put this at the top level to save some repetition: https://pre-commit.com/index.html#pre-commit-configyaml---top-level

Reply:
thanks, fixed